### DISCUSSION TITLE
Create POCO with only some fields

### DISCUSSION THREAD

**Discussion by kevinvenclovas** [linq2db/linq2db#3229] [Q&A] [answered]
Hi,

im using the .tt Template for Firebird to create my POCO's. My Database has many field that i didn't need for my application.
Is there an option create the POCO's only with the needed fields?

Databasetable: ID, Name, Age, Country, Lastlogin and many more
POCO should look like : ID, Name, Age

Can someone help me?

**Comment by sdanyliv** [comment, marked as answer]
Check documentation how to inject into generation process: https://linq2db.github.io/articles/T4.html#example-of-generation-process-customization

Then you can remove not needed columns:
```cs
foreach (var t in Tables.Values)
{
    t.Columns.Remove("Country");
    t.Columns.Remove("Lastlogin");
}
```
